### PR TITLE
#1396 add unit test for the csv content extraction helper

### PR DESCRIPTION
**Pull request by vrudas**
#1396

Hi @nya-elimuai, currently this pull request covers by tests only extraction of `Sound` and `Letter` objects from a CSV files. As other methods are more complex, what do you think about creation of a dedicated issues per remaining extraction methods in a [CsvContentExtractionHelper.java](https://github.com/elimu-ai/webapp/blob/4c56166ed221a9de41060bbfdda455e4a635016b/src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java) to make process more incremental(faster code reviews, less side effects, actual code base)?

### CODE REVIEW

**Comment by nya-elimuai** [comment]
> Hi @nya-elimuai, currently this pull request covers by tests only extraction of `Sound` and `Letter` objects from a CSV files. As other methods are more complex, what do you think about creation of a dedicated issues per remaining extraction methods in a [CsvContentExtractionHelper.java](https://github.com/elimu-ai/webapp/blob/4c56166ed221a9de41060bbfdda455e4a635016b/src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java) to make process more incremental(faster code reviews, less side effects, actual code base)?

@vrudas Thank you very much for adding this! And yes, creating new issues for the work with the more complex methods sounds like a good strategy 🙂

**Review by nya-elimuai — APPROVED** [review]

**Comment by vrudas** [comment]
> @vrudas Thank you very much for adding this! And yes, creating new issues for the work with the more complex methods sounds like a good strategy 🙂

Created an #1594, #1595, #1596, #1597, #1598 to cover remain methods by tests

File: src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java [hunk 1]
```diff
@@ -33,128 +41,19 @@
...
-        return letters;
-    }
-    
     /**
      * For information on how the CSV files were generated, see {@link LetterSoundCorrespondenceCsvExportController#handleRequest}.
      */
     public static List<LetterSoundCorrespondence> getLetterSoundCorrespondencesFromCsvBackup(File csvFile, LetterDao letterDao, SoundDao soundDao, LetterSoundCorrespondenceDao letterSoundCorrespondenceDao) {
         logger.info("getLetterSoundCorrespondencesFromCsvBackup");
-        
+
```
Comment (jo-elimu) [inline src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java:54]: Is this replacing spaces with tabs?

**Review by jo-elimu — APPROVED** [review]
Looks good to me! Thank you for contributing, @vrudas

File: src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java (hunk 1 above)
Comment (vrudas) [reply src/main/java/ai/elimu/util/csv/CsvContentExtractionHelper.java:54]: @jo-elimu removed all spaces/tabulation symbols from the line

**Pull request merged**